### PR TITLE
chore: 0.15.0 release proposal

### PR DESCRIPTION
**Pull request by dyladan**
#### :rocket: Enhancement
* Other
  * [#366](https://github.com/open-telemetry/opentelemetry-js-contrib/pull/366) Add automated release workflows ([@willarmiros](https://github.com/willarmiros))
  * [#391](https://github.com/open-telemetry/opentelemetry-js-contrib/pull/391) chore: adding meta package for web ([@obecny](https://github.com/obecny))
  * [#393](https://github.com/open-telemetry/opentelemetry-js-contrib/pull/393) feat: mysql instrumentation ([@dyladan](https://github.com/dyladan))
  * [#327](https://github.com/open-telemetry/opentelemetry-js-contrib/pull/327) feat: enable root span to contain route ([@DinaYakovlev](https://github.com/DinaYakovlev))
  * [#389](https://github.com/open-telemetry/opentelemetry-js-contrib/pull/389) feat: net module instrumentation ([@seemk](https://github.com/seemk))
  * [#379](https://github.com/open-telemetry/opentelemetry-js-contrib/pull/379) chore: creating meta package for default auto instrumentations for node ([@obecny](https://github.com/obecny))
  * [#380](https://github.com/open-telemetry/opentelemetry-js-contrib/pull/380) Moving Hapi Plugin to Instrumentation ([@obecny](https://github.com/obecny))
* `opentelemetry-host-metrics`
  * [#395](https://github.com/open-telemetry/opentelemetry-js-contrib/pull/395) chore: fixing broken links, updating to correct base url, replacing gitter with github discussions ([@obecny](https://github.com/obecny))

#### :house: Internal
* [#392](https://github.com/open-telemetry/opentelemetry-js-contrib/pull/392) chore: fixing broken links ([@obecny](https://github.com/obecny))

#### :memo: Documentation
* `opentelemetry-host-metrics`
  * [#373](https://github.com/open-telemetry/opentelemetry-js-contrib/pull/373) Update README.md ([@z1c0](https://github.com/z1c0))

#### Committers: 7
- Bartlomiej Obecny ([@obecny](https://github.com/obecny))
- Daniel Dyla ([@dyladan](https://github.com/dyladan))
- Dina.Yakovlev ([@DinaYakovlev](https://github.com/DinaYakovlev))
- Siim Kallas ([@seemk](https://github.com/seemk))
- Valentin Marchaud ([@vmarchaud](https://github.com/vmarchaud))
- William Armiros ([@willarmiros](https://github.com/willarmiros))
- Wolfgang Ziegler ([@z1c0](https://github.com/z1c0))

### CODE REVIEW

**Review by vmarchaud — APPROVED** [review]

File: plugins/node/opentelemetry-instrumentation-pg/package.json [hunk 1]
```diff
@@ -49,7 +49,7 @@
   "devDependencies": {
     "@opentelemetry/context-async-hooks": "0.18.0",
     "@opentelemetry/node": "0.18.0",
-    "@opentelemetry/test-utils": "^0.14.0",
+    "@opentelemetry/test-utils": "^0.15.0",
```
Comment (Flarna) [inline plugins/node/opentelemetry-instrumentation-pg/package.json:52]: same here

File: plugins/node/opentelemetry-plugin-mysql/package.json [hunk 2]
```diff
@@ -44,7 +44,7 @@
     "@opentelemetry/context-async-hooks": "0.18.0",
     "@opentelemetry/node": "0.18.0",
     "@opentelemetry/semantic-conventions": "0.18.0",
-    "@opentelemetry/test-utils": "^0.14.0",
+    "@opentelemetry/test-utils": "^0.15.0",
```
Comment (Flarna) [inline plugins/node/opentelemetry-plugin-mysql/package.json:47]: once again

**Review by Flarna — APPROVED** [review]

File: plugins/node/opentelemetry-instrumentation-mysql/package.json [hunk 3]
```diff
@@ -42,8 +42,8 @@
   },
   "devDependencies": {
     "@opentelemetry/context-async-hooks": "0.18.0",
+    "@opentelemetry/test-utils": "^0.15.0",
```
[the remark this answers is not in the document]
Comment (Flarna) [reply plugins/node/opentelemetry-instrumentation-mysql/package.json:45]: Actually I don't care that much, just noticed during scanning the files. Feel free to let the bots do the work.

**Comment by dyladan** [comment]
> Regarding the compile problem you have here. I think it's caused by the transitive dependency to `@hapi/podium` from `@hapi/hapi`. Version 4.1.2 was released recently. If you add `"@hapi/podium": "4.1.1"` in package.json problem should be gone.
> 
> Maybe updating all dependencies to latests solves the problem also.

Thanks. There is also a problem with the transitive dependencies hinted in https://github.com/open-telemetry/opentelemetry-js/issues/2065. The build fails because one of our dependencies (web) depends on another (tracing) with a `^` range. We also depend on a pinned version of `tracing`. This means tracing is installed twice with nesting in node_modules and typescript fails to install it. Fixing this requires merging the above PR.

**Review by jtmalinowski — CHANGES_REQUESTED** [review]
It will publish `propagator-jaeger@0.15.0`, even though it was already published `@0.18.2`. If we resolve the Hapi issue I can rebase https://github.com/open-telemetry/opentelemetry-js-contrib/pull/351 and we'll be good to merge it before publishing this release.

**Comment by dyladan** [comment]
> Regarding the compile problem you have here. I think it's caused by the transitive dependency to `@hapi/podium` from `@hapi/hapi`. Version 4.1.2 was released recently. If you add `"@hapi/podium": "4.1.1"` in package.json problem should be gone.
> 
> Maybe updating all dependencies to latests solves the problem also.

Your suggested fix doesn't work for me at least locally. Skipping lib checks in the tsconfig does, but that feels somehow wrong to me.

**Comment by vmarchaud** [comment]
> Skipping lib checks in the tsconfig does, but that feels somehow wrong to me.

We could only skip it for Hapi, sadly it sometimes happen and generally requires upstream dependencies to fix their types which can take some time. In our case that shouldn't impact too much i believe

**Comment by Flarna** [comment]
> Your suggested fix doesn't work for me at least locally

That's strange as I tried it locally and it worked for me.

**Comment by dyladan** [comment]
> > Your suggested fix doesn't work for me at least locally
> 
> That's strange as I tried it locally and it worked for me.

Can you send me the exact package.json? Or open a PR?

**Comment by Flarna** [comment]
Created https://github.com/open-telemetry/opentelemetry-js-contrib/pull/407 - let the CI decide

**Comment by dyladan** [comment]
Closing in favor of the bot version #409

**Pull request closed without merging**